optimum/exporters/tflite/base.py

```diff
@@ -40,6 +41,13 @@ class MissingMandatoryAxisDimension(ValueError):
     pass
 
 
+class QuantizationApproach(str, Enum):
+    INT8_DYNAMIC = "int8-dynamic"
+    INT8 = "int8"
+    INT8x16 = "int8x16"
+    FP16 = "fp16"
```

Comment:
Is there a reference in TFLite documentation where fp16 is considered as quantization? I don't think this is a common name.

Also, this could be confusing with the ONNX export that just uses `--fp16`.

Reply:
That's how they call it in TFLite => https://www.tensorflow.org/lite/performance/post_training_float16_quant?hl=en

But we could have a `--fp16` as well, which would be synonymous to `--quantization fp16`,

Reply:
Great if it's the terminology of TFLite let's keep it!